packages/libpangomm-2.48/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://www.gtkmm.org/
+TERMUX_PKG_DESCRIPTION="A C++ API for Pango"
+TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_MAINTAINER="@termux"
+_MAJOR_VERSION=2.50
```

Comment:
Same here, package is called libpangomm-2.48, but version is 2.50. Should package name be 2.50?